### ISSUE TITLE
[BUGNot Syncing

### ISSUE THREAD

**Issue by jatsc** [Chia-Network/chia-blockchain#8842]
The computer restarted
I try to reconnect again to chi, but I can not
It is trying to sycing
The message is Syncing 994,998/1,005,183
After some time it stops trying to sync and the message is Not synced

I have more computers exactle the same like this one and is working perfectly.

**Comment by SurvivingPsycho** [comment]
I'm having a similar problem except it won't even start to sync. I am running Windows 11 though so that may be the issue.

**Comment by loppefaaret** [comment]
yeah, it will keep incrementing as the blockchain moves along, any luck importing a DB from one of your other machines ?
